packages/modeling/src/operations/extrusions/extrudeFromSlices.js

```diff
@@ -22,7 +22,9 @@ const defaultCallback = (progress, index, base) => {
  *
  * @param {Object} options - options for extrude
  * @param {Integer} [options.numberOfSlices=2] the number of slices to be generated by the callback
- * @param {Boolean} [options.isCapped=true] the solid should have caps at both start and end
+ * @param {Boolean} [options.capStart=true] the solid should have a cap at the start
+ * @param {Boolean} [options.capEnd=true] the solid should have a cap at the end
+ * @param {Boolean} [options.close=false] the solid should have a closing section between start and end
```

Comment:
Good spot. I'm not 100% happy with the option names either. Here's a little more explanation.

The 'caps' are the start and the end parts of the extrusion. For example, the bottom and the top of the shapes from extrudeLinear. These 'caps' are not required, and are costly to calculate (booleans operations). Hollow shapes can be created by leaving the 'caps' off.

The torus() shape has an options called 'isCapped=true'. This can be set to false, which will produce a hollow shape for partial torus shapes.

As for 'close', this is kind of special. For a torus() shape, a full torus shape may produce a series of slices less then required to achieve full rotation. So there is a special piece of code that creates walls between end and start slices if required. Again, this little piece of functionality should be optional, as hollow shapes do not require a 'close' section.

Reply:
The option names could follow torus(), and become
- isStartCapped
- isEndCapped
- isClosed (between end and start cap)